ui/src/app/pages/analysis/interactive-notebook.tsx

```diff
@@ -101,20 +101,19 @@ export const InteractiveNotebook = fp.flow(withUrlParams(), withCurrentWorkspace
     }
 
     componentDidMount(): void {
-      workspacesApi().readOnlyNotebook(this.props.urlParams.ns,
-        this.props.urlParams.wsid, this.props.urlParams.nbName)
-        .then(html => {
-          this.setState({html: html.html});
-          this.loadNotebookActivityTracker();
-        });
+      const {ns, wsid, nbName} = this.props.urlParams;
```

Comment:
opt: revert? I like the change, though it's now unrelated to the PR

Reply:
keeping this